sources/packages/web/src/types/contracts/supporting-user/SupportingUserContracts.ts

```diff
@@ -1,7 +1,4 @@
 // Expected possible errors returned by the API while submitting supporting data.
-
-import { ContactInformation } from "@/types";
-
 // Not possible to find a Student Application with the data provided by the supporting user.
```

Comment:
Can we have an agreement to have these constants that are API-related under the `http` folder?
I would say that they will a possible part of the API response and because of that I would say that they can be added alongside the `DTOs`.

Reply:
moved

Reply:
Let's move the constants in another opportunity 😉 
@dheepak-aot @guru-aot @henryoforeh-aot, are we ok to have these constants alongside DTOs? It is just to keep then centralized and avoid declaring them again and again in different vue files.